internal/v1mcp/tools/cam.go

```diff
@@ -27,12 +27,15 @@ func toolCAMAwsAccountsList(client *v1client.V1ApiClient) mcpserver.ServerTool {
 			mcp.WithToolAnnotation(mcp.ToolAnnotation{
 				ReadOnlyHint: toPtr(true),
 			}),
-			mcp.WithNumber("top", mcp.Description("The number of records to display per page")),
+			mcp.WithString("top",
+				mcp.Description(tooldescriptions.DefaultTop),
+				mcp.Enum("25", "50", "100", "500", "1000", "5000"),
```

Comment:
❓ is it worth putting these enum values in the same structure as `cremTop`? so don't need to repeat these everywhere